service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadDocumentsControllerTest.java

```diff
@@ -21,13 +26,18 @@
 @OverrideAutoConfiguration(enabled = true)
 public class UploadDocumentsControllerTest extends AbstractControllerTest {
 
+    @MockBean
+    private UploadDocumentsService uploadDocumentsService;
+
     UploadDocumentsControllerTest() {
         super("upload-documents");
     }
 
     @Test
-    void shouldNotReturnErrorsWhenUploadedDocumentsAreValid() throws Exception {
+    void shouldNotReturnErrorsWhenUploadedDocumentsAreValid() {
         CaseDetails caseDetails = callbackRequest().getCaseDetails();
+        when(uploadDocumentsService.updateCaseDocuments(any(CaseData.class), any(CaseData.class)))
+            .thenReturn(caseDetails.getData());
```

Comment:
Should we mock this? I guess it's fine if it's tested in service.

Reply:
I am not sure that it should really, the main things that you want to mock in int tests are just APIs. The service should be able to just work as expected.

Reply:
just realised i don't need when thenReturn but without @MockBean of uploadDocumentsService test fails.